client/src/__tests__/container.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { mount, shallow } from 'enzyme';
+import Container from '../components/Container.jsx';
+
+configure({ adapter: new Adapter() });
+
+it('should render correctly with no props', () => {
+  const container = shallow(<Container />);
+  
+  expect(container).toMatchSnapshot();
+});
+
+it('should call getReviews during componentDidMount', () => {
+  const getReviews = jest.spyOn(Container.prototype, 'getReviews');
+  const container = mount(<Container />);
```

Comment:
When you run this aren't you supposed to MockRestore in case you have another test

Reply:
I am not sure 🤔 something I will have to explore more